Ports/pacman/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=pacman
+version=git
+workdir=pacman-master
+files="https://github.com/ebuc99/pacman/archive/refs/heads/master.tar.gz pacman-master.tar.gz 3016af69e168e0dd87a079bbfb35d995cf2e7bc1b710716c81ac99323bb4f7d5"
```

Comment:
Please lock this to a specific revision, even if it is seems unlikely that the project will get updated.

Reply:
Aye, good point. Should be fixed now. :)